composer.json

```diff
@@ -7,14 +7,13 @@
         "symfony/debug": "^2.6|^3.0",
         "symfony/process": "^2.6|^3.0",
         "react/event-loop": "^0.4",
-        "react/stream": "^0.4",
-        "react/http": "dev-master#cd15204bd15d106d7832c680e4fb0ca0ce2f5e30",
-        "react/socket-client": "^0.5.0",
-        "react/socket": "^0.5.0",
+        "react/stream": "^0.5",
+        "react/http": "dev-handle-psr7-responses",
+        "react/socket-client": "^0.6",
```

Comment:
Cannot do that, since `ReadableResourceStream` and `DuplexResourceStream` requires `react/stream: ^0.6`, but `react/socket-client: ^0.5` requires `react/stream: ^0.4.5`.